srcs/Http/HTTPRequest.cpp

```diff
@@ -42,6 +42,9 @@ std::string HTTPRequest::transfer_encoding() const {
 std::string HTTPRequest::request_body() const { return this->request_body_; }
 bool HTTPRequest::is_ready() const { return this->is_ready_; }
 
+void HTTPRequest::method(std::string method) { this->method_ = method; }
+void HTTPRequest::uri(std::string uri) { this->uri_ = uri; }
```

Comment:
```suggestion
void HTTPRequest::set_method(std::string method) { this->method_ = method; }
void HTTPRequest::set_uri(std::string uri) { this->uri_ = uri; }
```

セッターの命名規則はどうするか決めていましたっけ？もし決まっていなかったら、ゲッターは変数名、セッターはset_変数名がよさそうな気がするので、そうしたいです。以下はGoogle style guideからの引用です。

> アクセッサ(Getter)やミューテータ(Setter)は変数名のように名前を付けてもかまいません。 これらは、実際のメンバ変数と紐づけられますが、必須ではありません。 たとえば、int count()とvoid set_count(int count)です。

Reply:
賛成🙋

Reply:
修正しました！